public/app/features/dashboard-scene/utils/interactions.ts

```diff
@@ -225,6 +225,17 @@ export const DashboardInteractions = {
   copyImageUrlClicked: (properties?: Record<string, unknown>) => {
     reportDashboardInteraction('dashboard_image_url_copied', properties);
   },
+
+  // Dashboard library discard tracking
+  dashboardLibraryDiscarded: (
+    properties: DashboardLibraryTrackingInfo & {
+      timeBeforeDiscard?: number;
```

Comment:
this is not being used. why and how will you do this? a timer that starts when the dashboard is loaded? I find it a bit tricky